### PR TITLE
added name

### PR DESCRIPTION
**Pull request by aelkhou**

### CODE REVIEW

**Review by hamza-elkhoudiri — APPROVED** [review]
Looks good to me!

**Review by elkhoudh — APPROVED** [review]
LGTM!

File: Moar.js [hunk 1]
```diff
@@ -0,0 +1 @@
+new file
```
Comment (elkhoudh) [inline Moar.js:1]: @aelkhou did you mean to add this file?

Comment (aelkhou) [reply Moar.js:1]: No it was there already

**Pull request merged**